docs/sections/getting_started/basics.md

```diff
@@ -51,33 +52,49 @@ AiiDA ships with an interactive IPython shell that has many basic AiiDA classes
 To start the IPython shell, simply type in the terminal:
 
 ```{code-block} console
-
 $ verdi shell
-
 ```
 
+:::{margin} {{ python }} **Variables**
+
+Variables are a basic concept in Python and most programming languages.
+Understanding them is essential, so in case you are unfamiliar you can find a very short introduction [here](https://pythonbasics.org/variables/).
+
+:::
```

Comment:
I feel like this is a bit too much. If they don't know what a variable is, there is no way they will understand *anything* related to the tutorial or AiiDA.

Reply:
Maybe this is too much for this tutorial, but once it comes online for people to work on independently, I don't see why we can't add even these very basic concepts.